<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>566</width>
    <height>252</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="cedaTank" name="toolButton_tanque1">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>40</y>
      <width>271</width>
      <height>141</height>
     </rect>
    </property>
    <property name="text">
     <string>...</string>
    </property>
   </widget>
   <widget class="QLabel" name="toolButton_tanque1_2">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>20</y>
      <width>141</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string>toolButton_tanque1</string>
    </property>
   </widget>
   <widget class="Led" name="toolButton_s12">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>40</y>
      <width>33</width>
      <height>34</height>
     </rect>
    </property>
    <property name="text">
     <string>...</string>
    </property>
   </widget>
   <widget class="Led" name="toolButton_s11">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>150</y>
      <width>33</width>
      <height>34</height>
     </rect>
    </property>
    <property name="text">
     <string>...</string>
    </property>
   </widget>
   <widget class="Led" name="toolButton_v1">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>40</y>
      <width>33</width>
      <height>34</height>
     </rect>
    </property>
    <property name="text">
     <string>...</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>50</y>
      <width>58</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string>s12</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>160</y>
      <width>58</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string>s11</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>50</y>
      <width>58</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string>Valvula 1</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>566</width>
     <height>30</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <customwidgets>
  <customwidget>
   <class>cedaTank</class>
   <extends>QToolButton</extends>
   <header>cedatank.h</header>
  </customwidget>
  <customwidget>
   <class>Led</class>
   <extends>QToolButton</extends>
   <header>led.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
